tests/bud.bats

```diff
@@ -1821,3 +1821,11 @@ load helpers
 
   rm -f ${TESTSDIR}/bud/${target}/test*
 }
+
+@test "bud quite" {
```

Comment:
Typo: I think you mean "quiet". Please fix in lines 1825-1829.

Reply:
Get it. Thanks a lot.